Gopkg.toml

```diff
@@ -6,7 +6,7 @@ required = ["github.com/golang/protobuf/protoc-gen-go"]
 
 [[constraint]]
   name = "github.com/linkerd/linkerd2-proxy-api"
-  version = "v0.1.1"
+  revision = "61a94b46b5c576eb7c5c6ab6d9e74efc69fdc768" # TODO: switch back to stable
```

Comment:
Once linkerd/linkerd2-proxy-api#10 merges, I'll cut a new release in the linkerd2-proxy-api repo and switch this back to a stable dependency.